pkg/tfbridge/schema_test.go

```diff
@@ -2735,7 +2787,6 @@ func TestExtractSchemaInputsNestedMaxItemsOne(t *testing.T) {
 				"listObjects": resource.NewProperty([]resource.PropertyValue{
 					resource.NewProperty(resource.PropertyMap{
 						"__defaults": resource.NewProperty([]resource.PropertyValue{}),
-						"field1":     resource.NewProperty(false),
```

Comment:
What's happening here? Why this change?

Reply:
field1 is a boolean and we have logic which gets rid of default values for basic types when constructing inputs from outputs:
https://github.com/pulumi/pulumi-terraform-bridge/blob/18db7c57211af0eb7dfa26a70b145edc4d33a88f/pkg/tfbridge/schema.go#L1657

called from 

https://github.com/pulumi/pulumi-terraform-bridge/blob/18db7c57211af0eb7dfa26a70b145edc4d33a88f/pkg/tfbridge/schema.go#L1773

Seems like it was the original intention here.

Reply:
Gotcha. So this got fixed because the fix now propagates schema properly.